packages/mobile/src/account/InviteReview.tsx

```diff
@@ -48,7 +49,7 @@ interface DispatchProps {
 
 const mapStateToProps = (state: RootState): StateProps => ({
   inviteInProgress: state.invite.isSendingInvite,
-  dollarBalance: state.stableToken.balance,
+  dollarBalance: new BigNumber(state.stableToken.balance || 0),
```

Comment:
Could you check if this causes a bunch of rerenders? My understanding is that mapStateToProps is called every time any state changes - if it returns exactly the same thing, the component won't rerender (https://redux.js.org/faq/react-redux#why-is-my-component-re-rendering-too-often). I think this will always create a new different object which will probably mean this rerenders every time. In this case it may not matter, but just think we should be mindful of it

Reply:
Yes this re-renders everytime the redux store changes.

Reply:
Good catch, will fix